src/components/Dashboard/DashboardItem/AppItem.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+
+import { Launch, Info, Favorite, People, Apps } from "@material-ui/icons";
+import { IconButton, MenuItem } from "@material-ui/core";
+
+import { formatDate } from "@cyverse-de/ui-lib";
+
+import * as constants from "../constants";
+
+import ItemBase, { ItemAction } from "./ItemBase";
+
+class AppItem extends ItemBase {
+    constructor(props) {
+        super({
+            kind: constants.KIND_APPS,
+            content: props.content,
+            section: props.section,
+            height: props.height,
+            width: props.width,
+        });
+    }
+
+    static create(props) {
+        const item = new AppItem(props);
+        return item
+            .addActions([
+                <ItemAction
+                    arialLabel="favorite"
```

Comment:
Sorry, when I commented on `AnalysisItem`, it probably wasn't obvious I meant all of these item types had the same aria issue. These aria labels should be i18n